setup.py

```diff
@@ -1,21 +0,0 @@
-from setuptools import setup
```

Comment:
Please don't delete this file as it can be useful in legacy setups.

Reply:
Done !